### PR TITLE
[ICDS Dashboard] Infantometer and Stadiometer addition

### PR DESCRIPTION
**Pull request by challabeehyv**
https://app.asana.com/0/0/1139198196866667/f

##### SUMMARY
Infantometer and Stadiometer addition

##### FEATURE FLAG
Yes

### CODE REVIEW

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Challa could you address my comments on this Pr of same change: https://github.com/dimagi/commcare-hq/pull/25808

**Comment by challabeehyv** [comment]
> Challa could you address my comments on this Pr of same change: #25808

yes. Answered them

**Review by Rohit25negi — APPROVED** [review]
Good to merge as the build passes

File: custom/icds_reports/templates/icds_reports/icds_app/kpi.directive.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="row" ng-repeat="section in $ctrl.data track by $index">
-    <div class="col-md-6" ng-repeat="cell in section track by $index">
+    <div ng-if="((cell.label !== 'AWCs Reported Infantometer' && cell.label !== 'AWCs Reported Stadiometer' && cell.label !== 'Infantometer' && cell.label !== 'Stadiometer') || $ctrl.haveAccessToFeatures)" class="col-md-6" ng-repeat="cell in section track by $index">
```
Comment (czue) [inline custom/icds_reports/templates/icds_reports/icds_app/kpi.directive.html]: @challabeehyv I think this hard-coding of display logic against the labels is quite dangerous. I'd like to add a `show_in_kpis` field to this data model instead so it's more explicit and future proof. Can you tell me where I can go to test this is working properly? (e.g. where these are currently being hidden)?

Comment (challabeehyv) [reply custom/icds_reports/templates/icds_reports/icds_app/kpi.directive.html]: understood. I thought the feature flag is more like a temporary thing for testing purpose. I will make the change. 
https://india.commcarehq.org/a/icds-dashboard-qa/icds_dashboard/#/program_summary/awc_infrastructure
https://india.commcarehq.org/a/icds-dashboard-qa/icds_dashboard/#/awc_reports/awc_infrastructure?location_name=Basinapalli%20%5B28553020101%5D&location_id=0090df1176b2484d82b0d857c4697123&selectedLocationLevel=4
https://india.commcarehq.org/a/icds-dashboard-qa/icds_dashboard/#/awc_infrastructure/infantometer/map?location_name=C.K.Palli%20%5B285530201%5D&location_id=90be6e6a07f9433ab5fae105099d2e53&selectedLocationLevel=3
https://india.commcarehq.org/a/icds-dashboard-qa/icds_dashboard/#/awc_infrastructure/stadiometer/map?location_name=C.K.Palli%20%5B285530201%5D&location_id=90be6e6a07f9433ab5fae105099d2e53&selectedLocationLevel=3

Comment (czue) [reply custom/icds_reports/templates/icds_reports/icds_app/kpi.directive.html]: Sorry, you're saying this check will be deleted once the report is released? Then I feel less concerned (though if it's a simple change then great if you can make it).

Comment (challabeehyv) [reply custom/icds_reports/templates/icds_reports/icds_app/kpi.directive.html]: yes. This is for testing only. Anyways i will make the change

**Pull request merged**